docs/user/environment.md

```diff
@@ -0,0 +1,47 @@
+# Environment
+
+An environment in reinforcement learning is a task, or simulation, that an agent interacts with. Environments in garage are built on, and extends upon, Open AI's [Gym](https://gym.openai.com/) environments. One of the main differences is that garage uses [akro](https://akro.readthedocs.io/en/latest/) to describe input and output spaces, which is an extension of the gym.Space API.
```

Comment:
I think this should be reworded to "The garage environment API is close to Open AI's [Gym](https://gym.openai.com/) environment api." This is because we don't build on the the gym api, or actually directly inherit from it anymore.